plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -362,4 +368,18 @@ private Instance createInstance(String imageId,
             throw new MachineException(e);
         }
     }
+
+    String generateContainerName(String workspaceId, String displayName, String machineId) throws MachineException {
+        String workspaceName;
+        String userName = EnvironmentContext.getCurrent().getUser().getName();
+        try {
+            workspaceName = runtimeWorkspaceRegistry.get(workspaceId).getName();
+        } catch (NotFoundException e) {
+            throw new MachineException("Impossible to find running workspace for machine " + machineId, e);
+        }
+        workspaceName = workspaceName == null ? "" : workspaceName;
```

Comment:
Actually <code>workspaceName</code> is always not null.
<code>RuntimeWorkspaceRegistry</code> either returns <code>RuntimeWorkspace</code> instance or throws <code>NotFoundException</code>.